lib/fishjam/rpc_client.ex

```diff
@@ -0,0 +1,50 @@
+defmodule Fishjam.RPCClient do
+  @moduledoc """
+  This modules serves as simple RPC client to communicate with other nodes in cluster.
+  It utilizes the Enhanced version of Erlang `rpc` called `erpc`.
+
+  Enhanced version allows to  distinguish between returned value, raised exceptions, and other errors.
+  `erpc` also has better performance and scalability than the original rpc implementation.
+  """
+  require Logger
+
+  @doc """
+  Executes mfa on a remote node.
+  Function returns {:ok, result} tuple only if the execution succeeded.
+  In case of any exceptions we are catching them logging and returning simple :error atom.
+  """
+  @spec call(node(), module(), atom(), term(), timeout()) :: {:ok, term()} | :error
+  def call(node, module, function, args, timeout \\ :infinity) do
+    try do
+      {:ok, :erpc.call(node, module, function, args, timeout)}
+    rescue
+      e ->
+        Logger.warning("RPC call to node #{node} failed with exception: #{inspect(e)}")
+        :error
+    end
+  end
+
+  @doc """
+  Multicall to all nodes in cluster.
+  It filters out any errors or exceptions from return so you may end up with empty list.
+  """
+  @spec multicall(module(), atom(), term(), timeout()) :: list(term)
+  def multicall(module, function, args, timeout \\ :infinity) do
+    :erpc.multicall(nodes(), module, function, args, timeout)
+    |> handle_result()
```

Comment:
What do you think [about this point](https://github.com/christopheradams/elixir_style_guide?tab=readme-ov-file#avoid-single-pipelines) of elixir style guide?

Reply:
Ah that's actually good catch! Usually I use those checks in credo which should detect this situation:

1. https://github.com/rrrene/credo/blob/0058645812e29ac63a640905a2ad4958fc7bafd9/lib/credo/check/readability/single_pipe.ex#L5

2. https://github.com/rrrene/credo/blob/0058645812e29ac63a640905a2ad4958fc7bafd9/lib/credo/check/refactor/pipe_chain_start.ex#L5

Reply:
Although first one is something that I find myself controversial.
I really like pipe and even use it in just once 😄